serokell-style.md

```diff
@@ -418,19 +413,19 @@ putValueInState
 
 In other cases place `=` sign on the same line where function definition is.
 
-Use `() <$` to  ignore the result of function. It looks cool. If this is not possible
-due to `$` then use either `_ <-` or `void $`.
+Do not use `() <$` to ignore the result of function, because it harms readability.
```

Comment:
:unamused:

Reply:
The primary reason for `() <$` is that it's available from `Prelude` and `void` is not.

Reply:
We're advocating the use of `universum` anyway.

Reply:
I don't mind using `void`, but I find this rule too specific.

Reply:
Probably it will make sense to keep a brief version of the style guide and a longer version with things that didn't fit in the main version.